### PR TITLE
e2e: enable the url flag to control op result check

### PR DESCRIPTION
**Pull request by xhcao**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5990)

### CODE REVIEW

**Comment by xhcao** [comment]
@qjia7 @axinging @haoyunfeix @gyagp Please take a look, thank you.

**Comment by axinging** [comment]
LGTM

File: e2e/benchmarks/local-benchmark/index.html [hunk 1]
```diff
@@ -149,6 +149,15 @@ <h2>TensorFlow.js Model Benchmark</h2>
...
         }
+        if (tunableFlagsControllers['CHECK_COMPUTATION_FOR_ERRORS'] != null &&
+            urlState.has('CHECK_COMPUTATION_FOR_ERRORS')) {
+          const value = urlState.get('CHECK_COMPUTATION_FOR_ERRORS');
+          if (value === 'true') {
+            tunableFlagsControllers['CHECK_COMPUTATION_FOR_ERRORS'].setValue(true);
+          } else {
+            tunableFlagsControllers['CHECK_COMPUTATION_FOR_ERRORS'].setValue(false);
+          }
+        }
```
Comment (gyagp) [inline e2e/benchmarks/local-benchmark/index.html:160]: Can you merge the logic for WEBGL_USE_SHAPES_UNIFORMS, KEEP_INTERMEDIATE_TENSORS and this one?

Comment (xhcao) [reply e2e/benchmarks/local-benchmark/index.html]: Done

**Review by gyagp — COMMENTED** [review]
LGTM

**Comment by gyagp** [comment]
@pyu10055 @lina128 Could you please review this change? We found "CHECK_COMPUTATION_FOR_ERRORS" could impact the profiling result, so we hope to add a url flag to disable it in our auto test.

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5990#-:-Mt5adMIEDZfkeU1gMJ8:bb6ywmx)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @gyagp)

**Comment by xhcao** [comment]
Generally, when running a model in e2e, we will run 1 time for warm up, and run 50 times to compute the mean time for `Subsequent average`, then run 1 time for `profile time` and `kernel time`.
The below chart is the `inference times` for `Subsequent average`, 
image
We could see that the beginning runs is not steady, although we had warm up the model.  Run the AutoML Image on wasm backend with 1 profile times, the data is shown as below,

Subsequent average (50 runs)	8.4 ms
Best time	8.0 ms
Profile	12.8 ms
Conv2D	3.8 ms
DepthwiseConv2dNative	2.7 ms
Add	2.4 ms
Multiply	0.7 ms
Relu	0.4 ms
CropAndResize	0.2 ms
Cast	0.1 ms
Sub	0.1 ms
Mean	0.1 ms
AddN	0.1 ms
…	…

We could see: 1. `profile time` is much larger than `Subsequent average (50 runs)`; 2. The sum of all kernels' time is also larger than `Subsequent average (50 runs)`.
So I think it is reasonable that run profile many times and compute the mean time of each kernel. If we also runs 50 profiling times. The data is shown as below.
Subsequent average (50 runs)	8.3 ms
Best time	7.9 ms
Profile time	8.8 ms
Conv2D	3.0 ms
DepthwiseConv2dNative	1.9 ms
Add	1.2 ms
Multiply	0.6 ms
Relu	0.3 ms
CropAndResize	0.1 ms
Cast	0.1 ms
Sub	0.1 ms
Mean	0.1 ms
AddN	0.0 ms
…	…

**Pull request merged**